recipes/cleangeostreamr/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set name = "cleangeostreamr" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_url: https://codebase.helmholtz.cloud/department-computational-biology/software/cleangeostreamr.git
+  git_tag: v{{ version }}
+
+build:
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  script: R CMD INSTALL --build .
```

Should all R scripts be executable? Then 

```suggestion
  script: R CMD INSTALL --build . && chmod +x R/*.R && cp R/*.R ${PREFIX}/bin/
```

If it's not all `*.R` scripts than you may copy and chmod them individually. Maybe its also better to have it in a separate `build.sh` like here: https://github.com/bioconda/bioconda-recipes/blob/0db5639941324f8b3dc21fceb084f1d8284e8286/recipes/bioconductor-maaslin2/build.sh